packages/configs/default/index.json

```diff
@@ -7,7 +7,10 @@
       "@parcel/transformer-terser"
     ],
     "*.css": ["@parcel/transformer-css"],
-    "*.{htm,html}": ["@parcel/transformer-html"],
+    "*.{htm,html}": [
+      "@parcel/transformer-posthtml",
```

Comment:
I think the more aggressive thing should be first, let me know if that is not right

Reply:
Yeah, I think I got this backwards in the PostCSS transform

Reply:
Yeah posthtml/postcss should be first, in case they introduce new dependencies.

Reply:
Should I update postcss in this or separate?

Reply:
I moved `postcss` to be consistent.

Reply:
I ended up reverting the move of postcss, because it cause test failures. If it is suppose to be postcss first, we may have to look into why those tests were failing.